signer/migrations/0003__create_tables.sql

```diff
@@ -319,3 +319,31 @@ CREATE TABLE sbtc_signer.swept_deposits (
 -- unique per 'sweep_transaction_id'.
 CREATE UNIQUE INDEX uix_swept_deposits_req_txid_req_output_index_pkgd_txid
     ON sbtc_signer.swept_deposits(deposit_request_txid, deposit_request_output_index, sweep_transaction_txid);
+
+
+CREATE TABLE sbtc_signer.bitcoin_txs_sighashes (
+    txid BYTEA PRIMARY KEY,
+    chain_tip BYTEA NOT NULL,
+    prevout_txid BYTEA NOT NULL,
+    prevout_output_index INTEGER NOT NULL,
+    sighash BYTEA NOT NULL,
+    prevout_type sbtc_signer.prevout_type NOT NULL,
+    validation_result TEXT NOT NULL,
+    is_valid_tx BOOLEAN NOT NULL,
```

Comment:
Could you pad these tables out with some comments so it's clear what everything is?

Reply:
fixed in https://github.com/stacks-network/sbtc/pull/877/commits/0703ec69542199460a6c69a92ea352d432afc624